modules/local/validatemodel/tests/contrasts.yml

```diff
@@ -0,0 +1,10 @@
+contrasts:
```

Comment:
I recently added two contrasts.yml files into the test-datasets. You can use the ones there:
https://github.com/nf-core/test-datasets/blob/differentialabundance/testdata/SRP254919.contrasts.yaml
https://github.com/nf-core/test-datasets/blob/differentialabundance/testdata/MaxQuant_contrasts.yaml